jackson-apt/pom.xml

```diff
@@ -117,6 +117,8 @@
                 <configuration>
                     <moduleName>org.dominokit.jacksonapt.GwtJacksonApt</moduleName>
                     <moduleShortName>gwtjacksonapt</moduleShortName>
+                    <superSourceDirectory>../jackson-super/src/main/java</superSourceDirectory>
```

Comment:
Why do we need this?

Reply:
@vegegoku Well, I will be complete honest here - I just want to have all sources from jackson-super to be available in jackson-apt.jar for convenience in terms of packaging. Not sure if this should remain like that. Probably distributing jackson-super.jar is better...

Reply:
But jackson-apt can be used in both server side and client side and the sources are only required by the client side for GWT compilation, this might introduce issues when used in the server side..right?

Reply:
Yes, you are right. I will revert these changes and make sure everything is working as expected.

Reply:
Once this is fixed i will test in several projects i have before merging, i will do it as soon as you finish this.
Thank you

Reply:
@vegegoku Done.

Reply:
@tedynaidenov thank you for the update i will start testing very soon.